cmd/kubemark/app/hollow_node.go

```diff
@@ -264,7 +264,7 @@ func run(config *hollowNodeConfig) error {
 			runtimeService,
 			containerManager,
 		)
-		hollowKubelet.Run()
+		hollowKubelet.Run(context.TODO())
```

Comment:
Instead of `context.TODO()` here, change `run(config *hollowNodeConfig) error` to `run(ctx context.Context, config *hollowNodeConfig) error` and call it as `run(cmd.Context(), s)`.

Reply:
There's still a `context.TODO()` here.

Reply:
Still needs to be replaced:

```suggestion
		hollowKubelet.Run(ctx)
```